bom/pom.xml

```diff
@@ -38,6 +38,8 @@
     <influxdb.version>1.2.4-alpine</influxdb.version>
     <jackson.version>2.9.5</jackson.version>
     <java-base-image.name>openjdk:8u171-jre-slim</java-base-image.name>
+    <jaxb.api.version>2.3.0</jaxb.api.version>
+    <javax.annotation.api.version>1.3.2</javax.annotation.api.version>
```

Comment:
this dependency is not needed when I compile and run integration tests using Java 10